python/ray/autoscaler/_private/aws/cloudwatch/cloudwatch_helper.py

```diff
@@ -171,7 +172,7 @@ def ssm_install_cloudwatch_agent(self):
             "AWS-RunShellScript",
             parameters_run_shell,
         )
-        self._start_cloudwatch_agent(cwa_config_ssm_param_name)
+        self._restart_cloudwatch_agent(cwa_config_ssm_param_name)
```

Comment:
Thanks Patrick for fixing the issue of starting cloudwatch agent when agent is already configured through AMI and running. All changes looks good to me. I’ve tested the scenario of stopping the agent with no agent running here and everything works.